docs/ref/models/querysets.txt

```diff
@@ -2585,9 +2588,15 @@ Pass these flags as keyword arguments. For example, when using PostgreSQL::
     Execution time: 0.058 ms
 
 On some databases, flags may cause the query to be executed which could have
-adverse effects on your database. For example, PostgreSQL's ``ANALYZE`` flag
-could result in changes to data if there are triggers or if a function is
-called, even for a ``SELECT`` query.
+adverse effects on your database. For example, the ``ANALYZE`` flag supported
+by MariaDB, MySQL and PostgreSQL could result in changes to data if there are
```

Comment:
Please add serial comma: ``MariaDB, MySQL, and PostgreSQL``.

``could result`` -> ``results``?

Reply:
I think `could result` is more accurate here, it only results in changes if there is a trigger/function called that explicitly changes data. If we change it to `results` we should include that condition in the explanation IMO.

Reply:
Thanks for explanation.